contracts/NodeDelegator.sol

```diff
@@ -324,6 +321,40 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
         IOETH(asset).rebaseOptIn();
     }
 
+    /// @dev Approves the SSV Network contract to transfer SSV tokens for deposits
+    function approveSSV() external onlyLRTManager {
+        address SSV_TOKEN_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_TOKEN);
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+        if (SSV_TOKEN_ADDRESS == address(0)) revert NoSSVTokenConfig();
```

Comment:
I think we can get rid of the zero address checks in this contract code. Both for the SSV tokens and for WETH.

The solidity compiler automatically adds code to check that the destination address contains contract code before making a call. This catches calls to the zero address. (Solidity also automatically checks that the call was successful and did not revert. If doing a manual low level `call`, then you have to check for these things. But if solidity is generating the call like this, then we don't need to. 

<img width="469" alt="image" src="https://github.com/oplabs/primestaked-eth/assets/837/e5055098-2e9e-4dad-812a-21ea61c9247c">


Now there would be the case where we are passing an address as an argument and not calling it, but I think overall for the SSV_NETWORK_ADDRESS, I would prefer to just keep the code clean and simple. We can call the approve again if things were not setup.

Reply:
This makes sense. Thanks